AppServer/google/appengine/api/blobstore/blobstore_stub.py

```diff
@@ -289,8 +289,9 @@ def _Dynamic_CreateUploadURL(self, request, response, request_id):
                                   max_bytes_total,
                                   bucket_name)
 
-    protocol, host, _, _, _, _ = urlparse.urlparse(
-        self.request_data.get_request_url(request_id))
+    _, host, _, _, _, _ = urlparse.urlparse(
+      self.request_data.get_request_url(request_id))
```

Comment:
Can the call to get_scheme go inside of the get_request_url method? That way, anything else that calls get_request_url would also be fixed.